sdk/agrifood/agrifood-farming-rest/package.json

```diff
@@ -96,7 +96,7 @@
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^1.1.0",
     "@azure-tools/test-recorder": "^1.0.0",
-    "@microsoft/api-extractor": "7.13.2",
+    "@microsoft/api-extractor": "^7.18.11",
```

Comment:
Was there a discussion on using `^` vs `~` here? I wonder if the AE should be treated the same as the TS compiler where minor version releases could cause "breaking changes/significant api view changes"?

That being said, I do not think we will see any more significant changes in AE minor releases so we're probably fine. /cc @bterlson @xirzec

Reply:
Good question @deyaaeldeen ! Had not considered it to be honest. I do agree with you and Jeff that ^ is probably fine unless there's strong objections...

Reply:
Cool, let's run with this and we can always switch if needed